sklearnex/covariance/tests/test_incremental_covariance.py

```diff
@@ -37,6 +38,14 @@
 @pytest.mark.parametrize("dtype", [np.float32, np.float64])
 @pytest.mark.parametrize("assume_centered", [True, False])
 def test_sklearnex_partial_fit_on_gold_data(dataframe, queue, dtype, assume_centered):
+    try:
+        from dpctl import device_type
+
+        is_gpu = queue is not None and queue.sycl_device.device_type == device_type.gpu
+    except:
+        is_gpu = False
+    if assume_centered and is_gpu and not daal_check_version((2024, "P", 800)):
+        pytest.skip("There is a bug on oneDAL side")
```

Comment:
I'd leave something a bit more informative here. What kind of bug? incorrect results, memory corruption, seg fault?